extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -26,6 +27,9 @@ const EXPERIMENT = 'amp-sidebar';
 /** @const */
 const TAG = 'AmpSidebar';
 
+/** @const */
+const IOS_SAFARI_BOTTOMBAR_HEIGHT = '10vh';
```

Comment:
This seems very suspicious. I pretty sure iOS defines these things in plane pixels. No?